<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usermenue</class>
 <widget class="QMainWindow" name="usermenue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Menue</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 700 10pt &quot;Times New Roman&quot;;
background-color: rgb(165, 170, 169);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>142</y>
      <width>223</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Times New Roman&quot;;
background-color: rgb(189, 189, 189);</string>
    </property>
    <property name="text">
     <string>Applictions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>223</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Times New Roman&quot;;
background-color: rgb(189, 189, 189);</string>
    </property>
    <property name="text">
     <string>Main Menue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>223</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Times New Roman&quot;;
background-color: rgb(189, 189, 189);</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>223</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Times New Roman&quot;;
background-color: rgb(189, 189, 189);</string>
    </property>
    <property name="text">
     <string>Interviews</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>142</y>
      <width>223</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Times New Roman&quot;;
background-color: rgb(189, 189, 189);</string>
    </property>
    <property name="text">
     <string>Mentor Meeting</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>452</width>
      <height>136</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777214</width>
      <height>10000</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/eng.alahdal/Downloads/logo-1.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>469</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
